mlflow/server/js/src/experiment-tracking/components/CompareRunView.tsx

```diff
@@ -577,6 +588,17 @@ export class CompareRunView extends Component<CompareRunViewProps, CompareRunVie
           <Spacer size="lg" />
           {this.renderMetricTable(colWidth, experimentIds)}
         </CollapsibleSection>
+        <CollapsibleSection title={artifactsLabel}>
+          <Switch
+            label={diffOnlyLabel}
+            aria-label={[artifactsLabel, diffOnlyLabel].join(' - ')}
+            // @ts-expect-error TS(4111): Property 'onlyShowArtifactDiff' comes from an index ... Remove this comment to see the full error message
+            checked={this.state.onlyShowArtifactDiff}
+            onChange={(checked, e) => this.setState({ onlyShowArtifactDiff: checked })}
+          />
```

Comment:
does this do anything currently? it would be awesome to have image diffing but if it doesn't happen right now then let's just remove the switch haha